tests/admin/test_api_list_admin_users.py

```diff
@@ -70,7 +71,7 @@ def test_list_admin_users_api_resource_non_auth(self):
         expected_response = messages.AUTHORISATION_TOKEN_IS_MISSING
         actual_response = self.client.get("/admins")
 
-        self.assertEqual(401, actual_response.status_code)
+        self.assertEqual(HTTPStatus.UNAUTHORIZED, actual_response.status_code)
```

Comment:
replace it with HTTPStatus.UNAUTHORIZED.value

Reply:
Hey @devkapilbansal thanks for the review. I have made the changes. please review it again.